src/AbstractActionReader.ts

```diff
@@ -116,7 +116,7 @@ export abstract class AbstractActionReader {
       isNewBlock,
     }
 
-    return [this.currentBlockData, this.currentBlockMeta]
+    return [this.currentBlockData, this.currentBlockMeta, this.lastIrreversibleBlockNumber]
```

Comment:
is there a reason for returning an array instead of an object here?

Reply:
👍 seems like time for an object